polarCoordinateTraversal_old.m

```diff
@@ -106,12 +105,15 @@
 distance_from_circle_origin = (ray_start_x - circle_center_x)^2 + (ray_start_y - circle_center_y)^2;
 if distance_from_circle_origin > circle_max_radius^2
     current_voxel_ID_r = 1;
-    [tMaxR, tStepR] = radial_hit_old(ray_origin, ray_direction, ...
+    [tMaxR, ~] = radial_hit_old(ray_origin, ray_direction, ...
```

Comment:
Why ~? What happened to tStepR?

Reply:
We don't need tStepR for the initialization phase.